papaparse.js

```diff
@@ -297,8 +295,8 @@
 
 				if (!_input.fields)
 					_input.fields =  _input.data[0] instanceof Array
-									? _input.fields
-									: objectKeys(_input.data[0]);
+						? _input.fields
+						: objectKeys(_input.data[0]);
```

Comment:
We can relax the rules if we're not concerned with indentation like this. But it was easier to just fix this case.